### ISSUE TITLE
_has does not work with search params with dashes

### ISSUE THREAD

**Issue by ruscoder** [Aidbox/Issues#121]

**Comment by ruscoder** [comment]
Example:
```
GET /Patient?_has:EpisodeOfCare:patient:incoming-referral=43902d9b-d1a7-4bf9-a900-a414cb43ddd3
```

Output:
```
resourceType: OperationOutcome
status: 422
issue:
- {severity: error, code: invalid, diagnostics: 'No search parameter for EpisodeOfCare.EpisodeOfCare:incoming-referral'}
```

**Comment by ruscoder** [comment]
@niquola

**Comment by ruscoder** [comment]
```
GET /SearchParameter/EpisodeOfCare.incoming-referral
```

```
expression:
- [referralRequest]
meta: {lastUpdated: '2019-06-10T09:06:31.699Z', versionId: '0'}
name: incoming-referral
type: reference
resourceType: SearchParameter
source: code
module: fhir-4.0.0
id: EpisodeOfCare.incoming-referral
resource: {id: EpisodeOfCare, resourceType: Entity}
```

**Comment by niquola** [comment]
Ups - I will fix it

**Comment by niquola** [comment]
fixed in 0.4.6-SNAPSHOT
